packages/website/src/shared/lib/utilities/string.test.ts

```diff
@@ -0,0 +1,45 @@
+import { isSubstring } from "./string";
+
+describe("String Utilities", () => {
+  describe("isSubstring", () => {
+    it("handles without case-sensitivity", () => {
+      expect(isSubstring("", "")).toStrictEqual(true);
```

Comment:
nit. do we need toStrictEqual for booleans?

Reply:
Probably not. Same thing with strings, I don't think it does any harm, but if you feel strongly otherwise, we can definitely change it.

Reply:
agree it's harmless, no changes needed